integration-tests/ledger-signer.spec.ts

```diff
@@ -55,8 +55,8 @@ CONFIGS().forEach(({ lib, setup, rpc }) => {
       ).toBeInstanceOf(LedgerSigner);
     });
 
-    describe('Get the public key', () => {
-      it('Should get the correct public key and public key hash of the Ledger for tz1 curve and default path', async (done) => {
+    describe('Test retrieving the public key from the Ledger', () => {
+      it('As a User I want to verify that Ledger will provide correct public key and public key hash for tz1 curve and default path', async (done) => {
```

Comment:
Should we use the same format as the other files "Verify that..." without "As a User I want to"

Reply:
Updated the examples in this test set that were not "verify .." format